### DISCUSSION TITLE
regarding Swag Box For students

### DISCUSSION THREAD

**Discussion by Shanonbaker** [community/community#18844] [GitHub Education]
Hi! I am carl, I am a Web developer and a teacher. how can I get a swag box for my students as that will encourage them to be more Active?                    
And one more how do i upload my teacher id proof, i have a image but the option is only of  live snap{take a Snap} on github teacher tool box

**Comment by melvincwng** [comment]
Hi @libertycarl, first of all, welcome to GitHub Community.

For your question, you can refer to GitHub Education at this link for more info: https://education.github.com/

Hope this clarifies :)
